openedx/core/djangoapps/programs/tests/test_tasks.py

```diff
@@ -676,6 +676,27 @@ def setUp(self):
 
         self.inverted_programs = {self.course_key: [{'uuid': 1}, {'uuid': 2}]}
 
+    def _make_side_effect(self, side_effects):
```

Comment:
This utility function was right in the middle of all the unit tests, moved it above all the other test cases.